setuptools/dist.py

```diff
@@ -406,6 +406,7 @@ class Distribution(_Distribution):
         'long_description_content_type': None,
         'project_urls': dict,
         'provides_extras': set,
+        'license_files': list,
```

Comment:
I'm not sure this is necessary; why did `license_file` not require an entry here?

Reply:
Unless I'm missing a better way to do it, it seems like this is required for the list parser to actually extract and store the list into the metadata object. If I don't add this line, then the `license_files` member doesn't exist when `check_license()` is called.

I think since `license_file` is just a simple string value, it can use the `get()` method on the metadata dictionary, but if I inspect that for `license_files`, it just returns the raw string (e.g., `\nLICENSE-ABC\nLICENSE-XYZ\nLICENSE-PQR`)

Reply:
Got it. Thanks for the clarification.